client/src/components/MultiJurisdictionAudit/Setup/Review/useSampleSizes.ts

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import { api } from '../../../utilities'
+import { ISampleSizeOption } from '../../../../types'
+
+export interface IStringSampleSizeOptions {
+  [key: string]: {
+    size: string
+    type: string | null
+    prob: number | null
+  }[]
+}
+
+export interface ISampleSizeOptions {
+  [key: string]: ISampleSizeOption[]
+}
+
+export interface ISampleSizes {
+  [key: string]: string
+}
+
+const loadSampleSizes = async (
+  electionId: string
+): Promise<IStringSampleSizeOptions | null> => {
+  try {
+    const {
+      sampleSizes: options,
+    }: { sampleSizes: ISampleSizeOptions } = await api(
+      `/election/${electionId}/sample-sizes`
+    )
+    return Object.keys(options).reduce(
+      (a, contestId) => ({
+        ...a,
+        [contestId]: options[contestId].map(option => ({
+          ...option,
+          size: `${option.size}`,
+        })),
+      }),
+      {}
+    )
+  } catch (err) {
+    toast.error(err.message)
+    return null
+  }
+}
+
+const putSampleSizes = async (
+  electionId: string,
+  sampleSizes: ISampleSizes
+): Promise<boolean> => {
+  try {
+    await api(`/election/${electionId}/round`, {
+      method: 'POST',
+      body: JSON.stringify({
+        sampleSizes: {
+          // converts to number so it submits in the form: { [key: string]: number }
+          ...Object.keys(sampleSizes).reduce(
+            (a, v) => ({
+              ...a,
+              [v]: Number(sampleSizes[v]),
+            }),
+            {}
+          ),
+        },
+        roundNum: 1,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+    return true
+  } catch (err) {
+    toast.error(err.message)
+    return false
+  }
+}
+
+const useSampleSizes = (
+  electionId: string
+): [
+  IStringSampleSizeOptions | null,
+  (sampleSizes: ISampleSizes) => Promise<boolean>
+] => {
+  const [
+    sampleSizeOptions,
+    setSampleSizeOptions,
+  ] = useState<IStringSampleSizeOptions | null>(null)
+
+  const uploadSampleSizes = async (sizes: ISampleSizes): Promise<boolean> => {
```

Comment:
similar to comment above, i think this makes more sense as `createRound` as part of `useRounds`